server/src/models/v1/emailConfirmation.model.ts

```diff
@@ -0,0 +1,21 @@
+import { Table, Column, DataType, ForeignKey, CreatedAt, UpdatedAt, DeletedAt, BelongsTo, Model } from "sequelize-typescript";
+import Person from "./person.model";
+
+@Table({ tableName: "email_confirmations" })
```

Comment:
Should we rename this table according to the other ones?

```suggestion
@Table({ tableName: "dbo_email_confirmations" })
```